src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/CngKeyCreationOptions.cs

```diff
@@ -15,8 +15,21 @@ public enum CngKeyCreationOptions : int
         None = 0x00000000,
         MachineKey = 0x00000020,            // NCRYPT_MACHINE_KEY_FLAG
         OverwriteExistingKey = 0x00000080,  // NCRYPT_OVERWRITE_KEY_FLAG
+
+        /// <summary>
+        /// Indicates a key should be protected with Virtualization-based security (VBS).
+        /// </summary>
         PreferVbs = 0x00010000,             // NCRYPT_PREFER_VBS_FLAG
+
+        /// <summary>
+        /// Indicates a key must be protected with Virtualization-based security (VBS).
```

Comment:
For all of these, should it be "Virtualization-based security (VBS)" or "Virtualization-Based Security (VBS)"? (I feel like things being extracted to initialisms usually use title case... or rather "the extracted letters" case)

Reply:
I cased it the same way the win32 docs did, and the article on VBS does not title case it, except when actually used as a title.

* https://learn.microsoft.com/en-us/windows-hardware/design/device-experiences/oem-vbs
* https://learn.microsoft.com/en-us/windows/win32/api/ncrypt/nf-ncrypt-ncryptcreatepersistedkey